### ISSUE TITLE
Don't use $rootScope.metadata

### ISSUE THREAD

**Issue by mlynch** [driftyco/ionic-starter-push#2]
I think given the `$ionicUser` changes coming soon we will be rid of this but just wanted to add a placeholder here that if we do keep this metadata object it should be more "encapsulated"

**Comment by Fuiste** [comment]
Agreed. Once `identify()` is more integrated with the push library this shouldn't be needed at all.
